triage/labels.rst

```diff
@@ -10,405 +10,135 @@ Many labels are shared for both use cases, while some are dedicated
-
-Thus if an issue for a new feature is assigned for e.g., Python 3.8 but is not
-applied before Python 3.8.0 is released, this label should be updated to say
-``python-3.9`` as the version and drop ``python-3.8``.
-
-Priority
---------
-What is the severity and urgency?
-
-+------------------+--------------------------------------------------------+
-| Priority         | Description                                            |
-+==================+========================================================+
-| normal           | The default value for most issues filed.               |
-+------------------+--------------------------------------------------------+
-| deferred blocker | The issue will not hold up the next release, *n*. It   |
-|                  | will be promoted to a *release blocker* for the        |
-|                  | following release, *n+1*.                              |
-+------------------+--------------------------------------------------------+
-| release blocker  | The issue **must** be fixed before *any* release is    |
-|                  | made, e.g., will block the next release even if it is  |
-|                  | an alpha release.                                      |
-+------------------+--------------------------------------------------------+
-
-As a guideline, whether a bug is a *release blocker* for the current
-:ref:`release schedule <branchstatus>` is decided by the release manager.
-Triagers may recommend this priority and should notify the release manager by
-tagging them in a comment using ``@username``. If needed, consult the
-:ref:`release schedule <branchstatus>` and the release's associated PEP for the
-release manager's name.
-
-Keywords
---------
-Various informational flags about the issue. Multiple values are possible.
-
-+---------------+------------------------------------------------------------+
-|    Keyword    |                        Description                         |
-+===============+============================================================+
-| easy          | Fixing the issue should not take longer than a day for     |
-|               | someone new to contributing to Python to solve.            |
-+---------------+------------------------------------------------------------+
-
-Nosy List
----------
-A list of people who may be interested in an issue.
-
-This used to be a feature of the old issue tracker. On GitHub issues the
-same effect is achieved by tagging people in a comment using ``@username``.
-
-It is acceptable to tag someone to if you think the issue should be brought to
-their attention. Use the :ref:`experts` to know who wants to be added to the
-nosy list for issues targeting specific areas.
-
-If you want to subscribe yourself to an issue, click the *🔔 Subscribe*
-button in the sidebar. Similarly, if you were tagged by somebody else but
-decided this issue is not for you, you might click the *🔕 Unsubscribe*
-button in the sidebar.
```

Comment:
Part of this has been adapted and moved to the `issue-tracker.rst` page.

Reply:
Seems like most of the still relevant material is retained, but it is missing a bit of prior context about _how_ to tag people, especially given the audience there may include beginners not familiar with using GitHub. I'll make a suggestion there.